CHANGELOG.md

```diff
@@ -5,6 +5,16 @@ Note: this log contains only changes from knife-ec2 release 0.8.0 and later
 prior to release 0.8.0, please visit the [source repository](https://github.com/chef/knife-ec2/commits).
 
 
+## Latest Release 0.16.0 (yyyy-mm-dd)
```

Comment:
Date?

Reply:
Hi @tas50, as per comment of @NimishaS `Release date needs to be updated later on at the time of release.` https://github.com/chef/knife-ec2/pull/510#discussion-diff-146511732R8